src/runner.rs

```diff
@@ -2,13 +2,14 @@ use std::convert::From;
 use std::env;
 use std::error::Error;
 use std::fmt;
-use std::io::{Result as IoResult, Error as IoError};
+use std::io::{Result as IoResult, Error as IoError, ErrorKind};
 use std::path::{Path, PathBuf};
-use std::process::{Command, Stdio};
 use std::process;
-use mozprofile::profile::Profile;
+use std::process::{Command, Stdio};
+
 use mozprofile::prefdata::FIREFOX_PREFERENCES;
 use mozprofile::prefreader::PrefReaderError;
+use mozprofile::profile::Profile;
```

Comment:
Oh, I put in a separation line between standard libraries and third-party libraries, which lead to the confusion here.  We can put them all in the same block and sort them lexicographically.